Lib/fontgoggles/mac/mainWindow.py

```diff
@@ -559,13 +561,15 @@ def _gatherSidebarInfo(fonts):
             allAxes.append(font.axes)
             allScriptsAndLanguages.append(font.scripts)
             allStylisticSetNames.append(font.stylisticSetNames)
+            allInstances.extend(font.instances)
```

Comment:
Just re-reading this... if there is the same VF several times in the list of project fonts this will presumably add the instances several times as well... should to a merge down of the list.

Reply:
Improved this. I think the Popup widget had an automatic safeguard against identical items, but their order would be odd if adding the same fonts with intermitted other fonts. So unique list now, in order of instances read from fonts from the font list.